mocked-api/schemasData.ts

```diff
@@ -318,6 +326,36 @@ export const schemasData: { [key: string]: object } = {
       },
     },
   },
+  "google_pubsub_sink_0.1": {
+    type: "object",
+    additionalProperties: false,
+    required: [
+      "gcp_project_id",
+      "gcp_destination_name",
+      "gcp_service_account_key",
+    ],
+    properties: {
+      gcp_project_id: {
+        title: "Project Id",
+        description: "The Google Cloud PubSub Project Id",
+        type: "string",
+      },
+      gcp_destination_name: {
+        title: "Destination Name",
+        description:
+          "The Destination Name. For the consumer this will be the subscription name, while for the producer this will be the topic name.",
+        type: "string",
+      },
+      gcp_service_account_key: {
+        title: "Service Account Key",
+        description:
+          "The Service account key that can be used as credentials for the PubSub publisher/subscriber in base64 encoding.",
+        type: "string",
+        "x-group": "credentials",
```

Comment:
This is an interesting one. The back-end looks for a field of format `password` and not this `x-group: credentials` property. Meaning that the front-end considers it _secret_ whereas the back-end does not. However since "updates" to _secret_ fields uses the magic "empty object" pattern it is unlikely to affect operation.

@kostola FYI. It is likely [this](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/863) will show this field in the response as the [schema](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/main/processor/configuration/src/main/resources/schemas/actions/google_pubsub_sink_0.1.json) does not have a password field.